static/docs/command-reference/checkout.md

```diff
@@ -94,6 +83,13 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - ensures the file linking strategy (`reflink`, `hardlink`,
+  `symlink`, or `copy`) for all data files in the workspace is consistent with
+  the project's [`cache.type`](/doc/command-reference/config#cache). This is
+  achieved by restoring **all data files or a directories** referenced in
+  current DVC-files (regardless of whether they match a current DVC-file). This
+  means overwriting the file links or copies from cache to workspace.
```

Comment:
sentence is cryptic and complicated ... `overwriting ... copies from cache to workspace` - very hard to understand what does it mean

Reply:
How about just "Note that this overwrites the data files or directories in the workspace." I've changed it to that now but may have to reword all of this depending on Alex reply in https://github.com/iterative/dvc.org/pull/864#pullrequestreview-338379551 above.